libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Include filepaths for source and include.
+include(filePaths.cmake)
+
+# MQTT library target.
+add_library( mqtt
+             ${MQTT_SOURCES}
+             include/mqtt.h
+             include/mqtt_lightweight.h )
+
+# MQTT public include path.
+target_include_directories( mqtt PUBLIC ${MQTT_INCLUDE_PUBLIC_DIRS} )
+
+# Organization of MQTT in folders.
+set_property( TARGET mqtt PROPERTY FOLDER libraries/standard )
```

Comment:
(Minor:) Alternatively, this can use the specific `set_target_properties()` native function as `set_target_properties(mqtt PROPERTY FOLDER libraries/standard)`
https://cmake.org/cmake/help/v3.5/command/set_target_properties.html#command:set_target_properties

Reply:
oh cool